Editor/Resources/EditorWindow/GameLiftPlugin.uss

```diff
@@ -0,0 +1,59 @@
+/* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. */
+/* SPDX-License-Identifier: Apache-2.0 */
+
+:root {
+    height: 100%;
+    width: 100%;
+    
+    --color-menu-background: #333333;
```

Comment:
Instead of naming this based on where the background color is used, consider naming it to be descriptive of why it's used: `--color-background-accent` or `--color-background-alt`